sources/api/host-containers/src/main.rs

```diff
@@ -434,22 +434,48 @@ where
     Ok(())
 }
 
+fn is_restart_required(changed_settings: &str, container_name: &str) -> bool {
+    if changed_settings.is_empty() {
+        // it means that Bottlerocket is booting - all containers need to be started
+        return true;
+    }
+
+    let settings: Vec<&str> = changed_settings.split_whitespace().collect();
```

Comment:
I think it'd be good to do this split right after we get the value from the environment; we know any user of that data is going to want separated strings.